grafana-plugin/src/pages/integrations/Integrations.tsx

```diff
@@ -447,9 +445,7 @@ class Integrations extends React.Component<IntegrationsProps, IntegrationsState>
           </div>
         )}
       >
-        {({ openMenu }) => (
-          <HamburgerMenu openMenu={openMenu} listBorder={ACTIONS_LIST_BORDER} listWidth={ACTIONS_LIST_WIDTH} />
-        )}
+        {({ openMenu }) => <HamburgerMenu openMenu={openMenu} listBorder={2} listWidth={200} />}
```

Comment:
to be precise it's a kebab (see https://alvarotrigo.com/blog/assets/imgs/2021-12-14/hamburger-menu-types-examples.webp) =))